File: OPENAPI-REST-API/openapi-client/scalaz/src/main/scala/org/openapitools/client/api/PostWebsiteBuyIpRequest.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import PostWebsiteBuyIpRequest._

case class PostWebsiteBuyIpRequest (
  /* A map of IP addresses to their desired reverse DNS hostnames. */
  ips: Option[Map[String, String]])

object PostWebsiteBuyIpRequest {
  import DateTimeCodecs._

  implicit val PostWebsiteBuyIpRequestCodecJson: CodecJson[PostWebsiteBuyIpRequest] = CodecJson.derive[PostWebsiteBuyIpRequest]
  implicit val PostWebsiteBuyIpRequestDecoder: EntityDecoder[PostWebsiteBuyIpRequest] = jsonOf[PostWebsiteBuyIpRequest]
  implicit val PostWebsiteBuyIpRequestEncoder: EntityEncoder[PostWebsiteBuyIpRequest] = jsonEncoderOf[PostWebsiteBuyIpRequest]
}

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/PostWebsiteBuyIpRequest.js
goog.provide('API.Client.postWebsiteBuyIp_request');

/**
 * @record
 */
API.Client.PostWebsiteBuyIpRequest = function() {}

/**
 * A map of IP addresses to their desired reverse DNS hostnames.
 * @type {!Object<!string, string>}
 * @export
 */
API.Client.PostWebsiteBuyIpRequest.prototype.ips;

